pandas/tests/computation/test_eval.py

```diff
@@ -38,6 +39,23 @@
 _scalar_skip = 'in', 'not in'
 
 
+@pytest.fixture(params=[
+    pytest.mark.skipif(engine == 'numexpr' and not _USE_NUMEXPR,
+                       reason='numexpr enabled->{enabled}, '
+                              'installed->{installed}'.format(
+                            enabled=_USE_NUMEXPR,
+                            installed=_NUMEXPR_INSTALLED))(engine)
```

Comment:
Cosmetic + not worth changing, but for future pytesters - this sort of thing can go *in* the function